views/default/forms/admin/menu/save.php

```diff
@@ -7,11 +7,11 @@
 $num_featured_items = 6;
 
 // get site menu items
-$menu = elgg_get_config('menus');
-$menu = $menu['site'];
-$builder = new ElggMenuBuilder($menu);
-$menu = $builder->getMenu('name');
-$menu_items = $menu['default'];
+$menu = elgg()->menus->getMenu('site', [
```

Comment:
this menu gets prepared right??? you do not want this here... as we need an unprepared menu item... you should use the prepare flag